hasher-matcher-actioner/hmalib/common/s3_adapters.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import os
+import boto3
+import csv
+import codecs
+
+import typing as t
+
+from dataclasses import dataclass
+from hmalib import metrics
+from hmalib.common.logging import get_logger
```

Comment:
(ignorable/ymmv/there are actually reasons not to do this too): You can also do from .logging import get_logger